### DISCUSSION TITLE
Is there an example for a horizontal menu?

### DISCUSSION THREAD

**Discussion by scubakiz** [Blazored/Menu#26] [Q&A]
I don't see any "LayoutDirection" option.  Is it possible to have an expendable menu on a top bar (like on GitHub.com)?  Do you have an example?

**Comment by chrissainty** [comment]
There isn't anything built into the library that does this. You can achieve this by manipulating the CSS quite easily though. Adding this to your application style sheet should to the trick:

```css
.blazored-menu {
    display: flex;
}
```
